plugins/commitlint/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../../tsconfig.settings.json",
+  "references": [
+    {
+      "path": "../../lib/logger"
+    },
+    {
+      "path": "../../lib/base"
+    },
+    {
+      "path": "../../lib/schemas"
+    }
```

Comment:
**suggestion**: the reference to `schemas` isn't needed either

```suggestion
    }
```

Reply:
Sorry copy pasta and this is the one file I forgot to check